Core/Commands/Definitions/StaticResponseCommands.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using static Core.Commands.CommandUtils;
+
+namespace Core.Commands.Definitions
+{
+    public class StaticResponseCommands : ICommandCollection
+    {
+        // TODO moderators, operators
```

Comment:
I was unsure whether we want to keep those in the config or specify them e.g. in the database. Any thoughts?